### PR TITLE
Develop store resources

### PR DESCRIPTION
**Pull request by ghost**
**General**
- added a new $mobile breakpoint that covers 0 > $small
- moved links under h-wrap and h-med-wrap in a new group with link @includes
- subsequently removed specific paragraph>a definition in our-approach

**Cleaned a few things**
- new product-thumbnail.html and product-thumbnails.html to reflect project list
- removed some unused and duplicated archive and store .html files
- changed naming scheme to resources>resource, store>product, product-thumbnails>product-thumbnail to avoid overlapping the name "product"

**Image gallery**
- basically rewritten keeping the existing structure
- included new img.html
- merged gallery-img.html inside the loop

**Store + Resources**
- created store and resources scss files
- new layouts that reflect existing pages
- merged some edit done in Siteleaf to be able to test images and texts

### CODE REVIEW

**Comment by joshuarule** [comment]
Looks great. Nice work!
